src/ralph/virtual/migrations/0012_cloudimage.py

```diff
@@ -16,13 +16,9 @@ class Migration(migrations.Migration):
             name='CloudImage',
```

Comment:
You cannot change already merged migrations -- they are already applied somewhere. Just generate a new one.

Reply:
Done.